docs/spectrum1d.rst

```diff
@@ -115,9 +118,25 @@ any format, will default to the ``wcs1d-fits`` loader if the `~specutils.Spectru
 has a compatible WCS, and to ``tabular-fits`` otherwise, or if writing
 to another than the primary HDU (``hdu=0``) has been selected.
 For better control of the file type, the ``format`` parameter should be explicitly passed.
+Again, additional optional parameters are forwarded to the backend writing functions,
+which for the FITS writers is :meth:`astropy.io.fits.HDUList.writeto`.
 
 | More information on creating custom writers can be found in :ref:`custom_writer`.
 
+Metadata
+--------
+
+The :attr:`specutils.Spectrum1D.meta` attribute provides a dictionary to store
+additional information on the data, like origin, date and other circumstances.
+For spectra read from files containing header-like attributes like a FITS
+:class:`~astropy.io.fits.Header` or :attr:`astropy.table.Table.meta`,
+loaders are conventionally storing this in ``Spectrum1D.meta['header']``.
+
+Conversely the two provided FITS writers are saving the contents of any
+``Spectrum1D.meta['header']`` (which should be an :class:`astropy.io.fits.Header`
+or any object like a `dict` that can instantiate one) as the header of the
+:class:`~astropy.io.fits.hdu.PrimaryHDU`.
```

Comment:
```suggestion
The two provided FITS writers (`fits-tabular` and `wcs-fits`) save the contents of
``Spectrum1D.meta['header']`` (which should be an :class:`astropy.io.fits.Header`
or any object like a `dict` that can instantiate one) as the header of the
:class:`~astropy.io.fits.hdu.PrimaryHDU`.
```

Reply:
Please also add a link to the docstrings of the `tabular_fits` and the `wcs_fits` writers. I cant seem to find those currently included in the specutils docs. Maybe that's also a separate PR?

Reply:
I've added the suggestions; there are no API docs to any of the loaders one could link to, and I am afraid this will be non-trivial to amend since they are dynamically loaded through the unified IO framework.
I suppose one could build upon `_list_of_loaders` to then recursively call `Spectrum1D.read.help(format)` from its list entries to generate a doc page of all the readers (and writers)...
As mentioned above the `parsing_utils` functions are missing as well, but that should be more straightforward to fix; just don't know where to add the right hook for Sphinx. But those are more of interest for the _Custom Loader_ documentation. Definitely more than enough stuff for another PR, or two.